tests/tests/test_log.rs

```diff
@@ -161,6 +162,62 @@ fn concurrent_logging() {
     }
 }
 
+#[test]
+fn concurrent_logging_404() {
+    let config = ConfigBuilder::new()
+        .temporary(true)
+        .segment_mode(SegmentMode::Linear)
+        .io_buf_size(1000)
+        .flush_every_ms(Some(50))
+        .build();
+    let log_arc = Arc::new(Log::start_raw_log(config.clone()).unwrap());
+
+    static SHARED_COUNTER: AtomicUsize = AtomicUsize::new(0);
+    static SUCCESS_COUNTER: AtomicUsize = AtomicUsize::new(0);
+
+    let num_threads = 6;
+    let mut handles = Vec::new();
+    for t in 0..num_threads {
+        let log = log_arc.clone();
+        let h = thread::Builder::new()
+            .name(format!("t_{}", t))
+            .spawn(move || {
+                let iterations = 50_000;
```

Comment:
This is probably good to set to a const at the top of the function, maybe set to something more like 5000 to shave off a few minutes from the test run but still making it pretty easy to bump up in targeted testing